controllers/Index.php

```diff
@@ -429,7 +429,7 @@ protected function addPageSyntaxFields($formWidget, $page)
         foreach ($fields as $fieldCode => $fieldConfig) {
             if ($fieldConfig['type'] == 'fileupload') continue;
 
-            if ($fieldConfig['type'] == 'repeater') {
+            if ($fieldConfig['type'] == 'repeater' && !isset($fieldConfig['form'])) {
                 $fieldConfig['form']['fields'] = array_get($fieldConfig, 'fields', []);
```

Comment:
@Rike-cz change `$fieldConfig['form']['fields'] = array_get($fieldConfig, 'fields', []);` to `$fieldConfig['form']['fields'] = $this->makeConfig(array_get($fieldConfig, 'fields', []));` to fix the issue instead of the form attribute